java/memory/src/test/java/org/apache/arrow/memory/ITTestLargeArrowBuf.java

```diff
@@ -27,38 +32,45 @@
  *   main method.
  * </p>
  */
-public class TestLargeArrowBuf {
+public class ITTestLargeArrowBuf {
+  private static final Logger logger = LoggerFactory.getLogger(ITTestLargeArrowBuf.class);
```

Comment:
nit: according to the naming convention, a static final variable should be named with capital letters?

Reply:
logger is (in my experience anyways) an exception to the rule. Currently in arrow logger is used in 24 places and LOGGER in 15. I am happy with either way as long as we are consistent

Reply:
24 and 15. Sounds like we're super consistent. :D

Reply:
As Oscar Wilde says:

> Consistency is the hallmark of the unimaginative.

I have added https://issues.apache.org/jira/browse/ARROW-8739 to track